example-client/.env.client.example

```diff
@@ -0,0 +1,38 @@
+# Copy this file to a new file named '.env' to automatically load these
+# values as environment variables when running the client.
+# You will need to fill in the values based on which scenario you are
+# running against:
+# 1. A client testing against a Structure registered locally in the 
+#    Skatepark emulator.
+# 2. A client running against a Structure hosted in Griptape Cloud.
+
+# If the Structure is running in the Skatepark emulator, this value
+# will be the ID returned when building or registering the Structure
+# from the CLI.
+# In Griptape Cloud, it will be the ID of the structure as found on
+# https://cloud.griptape.ai/structures .
+GT_STRUCTURE_ID=YOUR_STRUCTURE_ID_HERE
+
+# If running against the Skatepark emulator, you will need to set
+# this environment variable (or put it in a .env file) to tell the
+# client which URL Skatepark is listening on. The default value is
```

Comment:
does this mean if they're happy with the default, they don't need to change it?

Reply:
When users point their client to Cloud they'll need to update this value to `https://cloud.griptape.ai`

Reply:
still an issue?